packaging/ubuntu-trusty/debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (quilt)
```

Comment:
Native is only for programs specifically developed for Debian:

> After squeeze, it should say either:
>  - 3.0 (native) for native Debian packages or
>  - 3.0 (quilt) for everything else.

See https://www.debian.org/doc/manuals/maint-guide/dother.en.html#sourcef